editor/plugins/org.fusesource.ide.wsdl2rest.ui/.gitignore

```diff
@@ -0,0 +1 @@
+lib/
```

Comment:
I think that there is no more lib folder in this plugin so this .gitignore file can be removed